src/pages/jinheeKim/Main/Main.js

```diff
@@ -0,0 +1,226 @@
+import React, {useState} from "react";
+import "../../../style/reset.scss";
+import "../../../style/common.scss";
+import "./Main.scss";
+import compass from "../assets/icon/compass.png";
+import heart from "../assets/icon/heart.png";
+import instagram from "../assets/icon/instagram.png";
+import search from "../assets/icon/search.png";
+import user from "../assets/icon/user.png";
+import userimg from "../assets/images/test.jpg";
+import feedimg from "../assets/images/test.jpg";
+import heart_click from "../assets/icon/heart_click.png";
+import share from "../assets/icon/share.png";
+import bookmark from "../assets/icon/bookmark-white.png";
+import chat from "../assets/icon/chat.png";
+import follower from "../assets/images/bf.jpg";
+import TextareaAutosize from 'react-textarea-autosize';
+
+const Main = () => {
+  const [comment, setComment] = useState('');
+  const saveComment = (event) => {
+    setComment(event.target.value);
+  };
+  const [commentArray, setCommentArray] = useState([]);
+
+  const onSubmit = event => {
+    event.preventDefault();
+    if (comment === '') {
+      return;
+    }
+    setCommentArray(commentList => [comment, ...commentList]);
```

Comment:
```suggestion
    setCommentArray(commentList => [ ...commentList,comment]);
```
이 두개의 차이점을 확실히 알고 넘어가주셨으면 좋겠습니다!!